software-modules/programming/sublime/sublime.sh

```diff
@@ -15,12 +15,18 @@
 #		Enya Quetzalli <equetzal@huronos.org>
 
 set -xe
-NAME=sublime
+NAME=sublime-text
 TARGET_DIR="/run/initramfs/memory/system/huronOS/software/programming/"
 
+## Add sublime keyring
+wget -qO - https://download.sublimetext.com/sublimehq-pub.gpg | apt-key add -
+
+## Add repository to deb source list
+echo "deb https://download.sublimetext.com/ apt/stable/" | tee /etc/apt/sources.list.d/sublime-text.list
+
 ## Install software
 apt update
-apt install --yes --no-install-recommends ./sublime-text_build-3211_amd64.deb
+apt install --yes --no-install-recommends "$NAME"
 apt autoremove --yes
 
 ## Prepare final files
```

Comment:
Line 27 does not work because of the name chage.
```txt
cp: cannot stat './sublime-text.desktop': No such file or directory
```
did you tested this PR?

Reply:
Yeah, I tested it hardcoding sublime-text on apt and it works